src/lib/OpenEXR/ImfInputFile.cpp

```diff
@@ -144,6 +145,10 @@ InputFile::InputFile (InputPartData* part)
     initialize ();
 }
 
+InputFile::~InputFile ()
```

Comment:
are you gonna put something in here? this might be just an =default in the header?

Reply:
oh, probably, I always get weird with virtual tables and force them to be in a specific translation unit. However, as soon as I remove the generic input file base class, will remove the vtable entirely, it is not needed / helpful.